pnpm-workspace.yaml

```diff
@@ -1,2 +1,3 @@
 packages:
-  - packages/*
+  - "package"
+  - "playground"
```

Comment:
Maybe just keep `packages/astro-remote` and move `playground` to `packages/playground`? Seems a bit cleaner.